tensorflow/python/keras/benchmarks/keras_examples_benchmarks/bidirectional_lstm_benchmark_test.py

```diff
@@ -21,14 +23,14 @@
 
 import tensorflow as tf
 
-from tensorflow.python.keras.benchmarks import benchmark_util
 from tensorflow.python.platform import benchmark
+from tensorflow.python.keras.benchmarks import benchmark_util
 
 _MAX_FEATURE = 20000
 _MAX_LEN = 200
 
 
-class KerasExamplesBenchmark(
+class BidirectionalLSTMBenchmark(
     six.with_metaclass(benchmark.ParameterizedBenchmark, tf.test.Benchmark)):
   """Required Arguments for measure_performance:
```

Comment:
nit: Here should be the doc string for this benchmark class. 

Is it better to place the docstring of arguments in the benchmark method?

Reply:
Yes, I will change it.